src/fprime/util/cookiecutter_wrapper.py

```diff
@@ -178,9 +178,35 @@ def new_deployment(build: Build, parsed_args: "argparse.Namespace"):
             + "/../cookiecutter_templates/cookiecutter-fprime-deployment"
         )
         print("[INFO] Cookiecutter: using builtin template for new deployment")
+
+    # Determine the include path prefix based on the current directory
+    proj_root = Path(build.get_settings("project_root", None)).resolve()
```

Comment:
Could we spin this whole block of code in a utility function such as `get_relative_include_path()` or something? Just to make the entire `new_deployment` more logical / easy to read to read

Reply:
Sure!